compiler/rustc_middle/src/mir/interpret/value.rs

```diff
@@ -152,8 +152,8 @@ impl<Tag: Provenance> fmt::Debug for Scalar<Tag> {
 impl<Tag: Provenance> fmt::Display for Scalar<Tag> {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         match self {
-            Scalar::Ptr(ptr, _size) => write!(f, "pointer to {:?}", ptr),
```

Comment:
This causes the ui test fallout. Please make sure the code that prints ui tests uses the debug formatting then

Reply:
:+1: